services/job_subscriber_test.go

```diff
@@ -158,20 +159,32 @@ func TestJobSubscriber_OnNewHead_OnlyResumePendingConfirmations(t *testing.T) {
 	t.Parallel()
 
 	block := cltest.NewBlockHeader(10)
+	prettyLabel := func(archived bool, rs models.RunStatus) string {
```

Comment:
Curious, any significant difference between naming the function and assigning an anonymous function to a variable?

Reply:
Don't believe so, it's just stored in an intermediary variable: https://yourbasic.org/golang/anonymous-function-literal-lambda-closure/

Reply:
Sorry, I misunderstood the question. You can't have a named function inside another function. With this anonymous function, it's scoped just to this test.

![image](https://user-images.githubusercontent.com/635121/54313459-c8aec100-45af-11e9-8067-3552e68be70a.png)